<?xml version="1.0" ?>
<!-- =================================================================================== -->
<!-- |    This document was autogenerated by xacro from manipulator_gazebo/urdf/manipulator.urdf.xacro | -->
<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
<!-- =================================================================================== -->
<robot name="manipulator">
  <link name="world"/>
  <joint name="fixed_joint" type="fixed">
    <parent link="world"/>
    <child link="base_link"/>
    <origin rpy="0 0 0" xyz="0 0 0"/>
  </joint>
  <link name="base_link">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="/home/ekumen/colcon_ws/src/manipulator_sim/manipulator_gazebo/model/manipulator/base.stl"/>
      </geometry>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="/home/ekumen/colcon_ws/src/manipulator_sim/manipulator_gazebo/model/manipulator/base.stl"/>
      </geometry>
    </collision>
    <inertial>
      <mass value="1.0"/>
      <origin xyz="0 0 0"/>
      <inertia ixx="1.6666666666666667e-05" ixy="0.0" ixz="0.0" iyy="1.6666666666666667e-05" iyz="0.0" izz="1.6666666666666667e-05"/>
    </inertial>
  </link>
  <gazebo reference="base_link">
    <material>Gazebo/Red</material>
  </gazebo>
  <joint name="joint_1" type="continuous">
    <parent link="base_link"/>
    <child link="link_1"/>
    <axis xyz="0 0 1"/>
    <origin rpy="0 0 0" xyz="0 0 0.03"/>
  </joint>
  <link name="link_1">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="/home/ekumen/colcon_ws/src/manipulator_sim/manipulator_gazebo/model/manipulator/link_1.stl"/>
      </geometry>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="/home/ekumen/colcon_ws/src/manipulator_sim/manipulator_gazebo/model/manipulator/link_1.stl"/>
      </geometry>
    </collision>
    <inertial>
      <mass value="1.0"/>
      <origin xyz="0 0 0"/>
      <inertia ixx="1.6666666666666667e-05" ixy="0.0" ixz="0.0" iyy="1.6666666666666667e-05" iyz="0.0" izz="1.6666666666666667e-05"/>
    </inertial>
  </link>
  <gazebo reference="link_1">
    <material>Gazebo/White</material>
  </gazebo>
  <joint name="joint_2" type="revolute">
    <parent link="link_1"/>
    <child link="link_2"/>
    <axis xyz="0 0 1"/>
    <limit effort="1000.0" lower="0.7853981633974483" upper="1.5707963267948966" velocity="1.0"/>
    <origin rpy="1.5707963267948966 0 0" xyz="0 0 0.03"/>
  </joint>
  <link name="link_2">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="/home/ekumen/colcon_ws/src/manipulator_sim/manipulator_gazebo/model/manipulator/link_2.stl"/>
      </geometry>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="/home/ekumen/colcon_ws/src/manipulator_sim/manipulator_gazebo/model/manipulator/link_2.stl"/>
      </geometry>
    </collision>
    <inertial>
      <mass value="1.0"/>
      <origin xyz="0 0 0"/>
      <inertia ixx="1.6666666666666667e-05" ixy="0.0" ixz="0.0" iyy="1.6666666666666667e-05" iyz="0.0" izz="1.6666666666666667e-05"/>
    </inertial>
  </link>
  <gazebo reference="link_2">
    <material>Gazebo/Blue</material>
  </gazebo>
  <joint name="joint_3" type="revolute">
    <parent link="link_2"/>
    <child link="link_3"/>
    <axis xyz="0 0 1"/>
    <limit effort="1000.0" lower="-3.141592653589793" upper="1.5707963267948966" velocity="1.0"/>
    <origin rpy="0 0 0" xyz="0.25 0 0"/>
  </joint>
  <link name="link_3">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="/home/ekumen/colcon_ws/src/manipulator_sim/manipulator_gazebo/model/manipulator/link_3.stl"/>
      </geometry>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="/home/ekumen/colcon_ws/src/manipulator_sim/manipulator_gazebo/model/manipulator/link_3.stl"/>
      </geometry>
    </collision>
    <inertial>
      <mass value="1.0"/>
      <origin xyz="0 0 0"/>
      <inertia ixx="1.6666666666666667e-05" ixy="0.0" ixz="0.0" iyy="1.6666666666666667e-05" iyz="0.0" izz="1.6666666666666667e-05"/>
    </inertial>
  </link>
  <gazebo reference="link_3">
    <material>Gazebo/Green</material>
  </gazebo>
</robot>
